From 53045d5dcdce3988e2586cb315b35e6a675a8152 Mon Sep 17 00:00:00 2001
From: Panu Matilainen <pmatilai@redhat.com>
Date: Wed, 30 Jun 2010 11:57:17 +0300
Subject: [PATCH 1/2] Fix potential getOutputFrom() error on font provide extraction (RhBug:609117)
 - if fonts are detected in the package being built but fc-query isn't
   present, the script exited without consuming stdin which can break
   getOutputFrom()

---
 scripts/fontconfig.prov |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/scripts/fontconfig.prov b/scripts/fontconfig.prov
index 8638680..594851d 100755
--- a/scripts/fontconfig.prov
+++ b/scripts/fontconfig.prov
@@ -12,7 +12,10 @@
 
 fcquery=/usr/bin/fc-query
 
-[ -x $fcquery ] || exit 0
+if [ ! -x $fcquery ]; then
+    cat > /dev/null
+    exit 0
+fi
 
 # filter out anything outside main fontconfig path
 grep /usr/share/fonts/ |
-- 
1.7.0.1

